src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+body,
+header,
+a,
+img,
+div,
+nav,
+ul,
+li {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
there is no necessary to use styles like this globally
use it directly to item

Reply:
![image](https://user-images.githubusercontent.com/97075591/204393601-9190c5f7-7584-46c3-97aa-8b5b694fb284.png)
So what i need to do?